docs/content/getting-started/update-tuples.mdx

```diff
@@ -199,11 +200,113 @@ Assume that you want to delete user `user:anne`'s `reader` relationship with obj
   ]}
 />
 
+### 04. Writing and deleting relationship tuples in the same request
+
+You can combine both writes and deletes in a single transactional API request. This is useful when you need to update multiple relationships atomically. All operations in the request will either succeed together or fail together.
+
+The Write API allows you to send up to <ConfigValue name="MAX_TUPLES_PER_WRITE" /> unique tuples in the request. (This limit applies to the sum of both writes and deletes in that request).
+
+For example, you might want to remove `user:anne` as a `writer` of `document:Z` while simultaneously updating `user:anne` as an `reader` of `document:Z`:
+
+<WriteRequestViewer
+  relationshipTuples={[
+    {
+      user: 'user:anne',
+      relation: 'reader',
+      object: 'document:Z'
+    },
+  ]}
+  deleteRelationshipTuples={[
+    {
+      user: 'user:anne',
+      relation: 'writer',
+      object: 'document:Z'
+    },
+  ]}
+  skipSetup={true}
+  allowedLanguages={[
+    SupportedLanguage.JS_SDK,
+    SupportedLanguage.GO_SDK,
+    SupportedLanguage.DOTNET_SDK,
+    SupportedLanguage.PYTHON_SDK,
+    SupportedLanguage.JAVA_SDK,
+    SupportedLanguage.CLI,
+    SupportedLanguage.CURL,
+  ]}
+/>
+
+This approach ensures that both operations succeed or fail together, maintaining transactional data consistency.
+
+
+:::note
+When using the Write API, you cannot include the same tuple (same user, relation, and object) in the writes or deletes arrays within a single request. The API will return an error with code `cannot_allow_duplicate_tuples_in_one_request` if detected.
+:::
+
+### 05. Ignoring duplicate tuples
```

Comment:
the delete use case is not "ignoring duplicate tuples", right? should we change the title? Add another section?

Reply:
Renamed title to "Ignoring duplicate or missing tuples"